lib/std/testing.zig

```diff
@@ -450,6 +450,33 @@ pub fn buildExe(zigexec: []const u8, zigfile: []const u8, binfile: []const u8) !
     try expectEqual(ret_val, .{ .Exited = 0 });
 }
 
+/// Spawns a zig build runner process 'zigexec build subcmd' and
```

Comment:
This does not mention what `cwd` and `lib_dir` are used for.
For `test` commands the structure is always the same, but for `cwd` and `lib_dir` we have significant variability or what do you think?
The user could also set the `lib_dir` itself, so this sounds not too convincing yet to me.

The main advantage to justify `runZigBuild` I see from offering `cwd`.

Reply:
I m not sure I understand your question correctly. Are you talking about lack of doc comments explaining what does what?

For now, cwd is used to mention the directory in which ``zig build`` will run, this is absolutely needed. lib_dir is actually used to specify the path to the zig library which will be used. This is needed because we cannot assume which zig compiler is being used to perform the test, so its better to default it to use the local stdlib present in the source directory.

Reply:
Ah, sounds like a frequent operation to test or ensure compatibility.
Please describe both very briefly in the doc comment.

Reply:
Hopefully clear enough now, not putting use case info into doc comments because thats not what they are for, generally.

Reply:
@matu3ba I would like you refrain from reviewing code that is not directly related to changes that you are responsible for. Generally, your code reviews have been confusing, and thereby misleading to newcomers. Please leave nearly all code reviews to others.

Reply:
ok. no more reviews from me then.